server/marlin/marlin_kernels/__init__.pyi

```diff
@@ -19,6 +19,23 @@ def gptq_marlin_gemm(
     """
     ...
 
+def gptq_marlin_24_gemm(
```

Comment:
Are we running a fork of marlin ?
Shouldn't we use the classic makefile + commit approach instead (or already made releases if possible) ?

Reply:
These come from VLLM. We could also directly import them from VLLM if we prefer.